pkg/providers/tinkerbell/config/template-cp.yaml

```diff
@@ -66,8 +66,6 @@ spec:
                 value: "true"
               - name: port
                 value: "6443"
-              - name: vip_interface
-                value: eno2
```

Comment:
Have we verified that we are using the latest kubevip image now? This won't work without that

Reply:
yes, our CLI is using the latest kubeVip image.